### PR TITLE
DOCS-5147 Android Log Collection Edit

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds a tab for the AP1 site region.

### Motivation

DOCS-5147

## Preview

https://docs-staging.datadoghq.com/alai97/log-collection-small-ap1-doc-update/logs/log_collection/android

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [x] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: docs/log_collection.md [hunk 1]
```diff
@@ -18,209 +20,254 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
     }
     ```
 
-2. Initialize the library with your application context, tracking consent, and the [Datadog client token][2] and Application ID generated when you create a new RUM application in the Datadog UI (see [Getting Started with Android RUM Collection][6] for more information). For security reasons, you must use a client token: you cannot use [Datadog API keys][3] to configure the `dd-sdk-android` library as they would be exposed client-side in the Android application APK byte code. For more information about setting up a client token, see the [client token documentation][2]. The `APP_VARIANT_NAME` specifies the variant of the application that generates data. 
+2. Initialize the library with your application context, tracking consent, as well as the [Datadog client token][2] and Application ID provided when you [created a RUM application][6]. For security reasons, you must use a client token: you cannot use [Datadog API keys][3] to configure the `dd-sdk-android` library as they would be exposed client-side in the Android application APK byte code.
```
Comment (rtrieu) [inline docs/log_collection.md:23]:
```suggestion
2. Initialize the library with your application context, tracking consent, as well as the [Datadog client token][2] and Application ID provided when you [created a RUM application][6]. For security reasons, you must use a client token; you cannot use [Datadog API keys][3] to configure the `dd-sdk-android` library, as they would be exposed client-side in the Android application APK byte code.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Added a minor suggestion, but otherwise looks good.

File: docs/log_collection.md [hunk 2]
```diff
@@ -18,209 +20,254 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
...
+   ```java
+       public class SampleApplication extends Application {
+           @Override
+           public void onCreate() {
+               super.onCreate();
+               Configuration configuration =
+                       new Configuration.Builder(true, true, true, true)
+                               .build();
+               Credentials credentials = new Credentials( < CLIENT_TOKEN >, <ENV_NAME >, <APP_VARIANT_NAME >, <
+               APPLICATION_ID >);
```
Comment (0xnm) [inline docs/log_collection.md:58]:
```suggestion
               Credentials credentials = new Credentials(<CLIENT_TOKEN>, <ENV_NAME>, <APP_VARIANT_NAME>, <APPLICATION_ID>);
```

File: docs/log_collection.md [hunk 3]
```diff
@@ -18,209 +20,254 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
...
+       public class SampleApplication extends Application {
+           @Override
+           public void onCreate() {
+               super.onCreate();
+               Configuration configuration =
+                       new Configuration.Builder(true, true, true, true)
+                               .useSite(DatadogSite.EU1)
+                               .build();
+               Credentials credentials = new Credentials( < CLIENT_TOKEN >, <ENV_NAME >, <APP_VARIANT_NAME >, <
+               APPLICATION_ID >);
```
Comment (0xnm) [inline docs/log_collection.md:99]:
```suggestion
               Credentials credentials = new Credentials(<CLIENT_TOKEN>, <ENV_NAME>, <APP_VARIANT_NAME>, <APPLICATION_ID>);
```

File: docs/log_collection.md [hunk 4]
```diff
@@ -18,209 +20,254 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
...
+       public class SampleApplication extends Application {
+           @Override
+           public void onCreate() {
+               super.onCreate();
+               Configuration configuration =
+                       new Configuration.Builder(true, true, true, true)
+                               .useSite(DatadogSite.US3)
+                               .build();
+               Credentials credentials = new Credentials( < CLIENT_TOKEN >, <ENV_NAME >, <APP_VARIANT_NAME >, <
+               APPLICATION_ID >);
```
Comment (0xnm) [inline docs/log_collection.md:140]:
```suggestion
               Credentials credentials = new Credentials(<CLIENT_TOKEN>, <ENV_NAME>, <APP_VARIANT_NAME>, <APPLICATION_ID>);
```

File: docs/log_collection.md [hunk 5]
```diff
@@ -18,209 +20,254 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
...
+       public class SampleApplication extends Application {
+           @Override
+           public void onCreate() {
+               super.onCreate();
+               Configuration configuration =
+                       new Configuration.Builder(true, true, true, true)
+                               .useSite(DatadogSite.US5)
+                               .build();
+               Credentials credentials = new Credentials( < CLIENT_TOKEN >, <ENV_NAME >, <APP_VARIANT_NAME >, <
+               APPLICATION_ID >);
```
Comment (0xnm) [inline docs/log_collection.md:181]:
```suggestion
               Credentials credentials = new Credentials(<CLIENT_TOKEN>, <ENV_NAME>, <APP_VARIANT_NAME>, <APPLICATION_ID>);
```

File: docs/log_collection.md [hunk 6]
```diff
@@ -18,209 +20,254 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
...
+       public class SampleApplication extends Application {
+           @Override
+           public void onCreate() {
+               super.onCreate();
+               Configuration configuration =
+                       new Configuration.Builder(true, true, true, true)
+                               .useSite(DatadogSite.US1_FED)
+                               .build();
+               Credentials credentials = new Credentials( < CLIENT_TOKEN >, <ENV_NAME >, <APP_VARIANT_NAME >, <
+               APPLICATION_ID >);
```
Comment (0xnm) [inline docs/log_collection.md:222]:
```suggestion
               Credentials credentials = new Credentials(<CLIENT_TOKEN>, <ENV_NAME>, <APP_VARIANT_NAME>, <APPLICATION_ID>);
```

File: docs/log_collection.md [hunk 7]
```diff
@@ -18,209 +20,254 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
...
+       public class SampleApplication extends Application {
+           @Override
+           public void onCreate() {
+               super.onCreate();
+               Configuration configuration =
+                       new Configuration.Builder(true, true, true, true)
+                               .useSite(DatadogSite.AP1)
+                               .build();
+               Credentials credentials = new Credentials( < CLIENT_TOKEN >, <ENV_NAME >, <APP_VARIANT_NAME >, <
+               APPLICATION_ID >);
```
Comment (0xnm) [inline docs/log_collection.md:263]:
```suggestion
               Credentials credentials = new Credentials(<CLIENT_TOKEN>, <ENV_NAME>, <APP_VARIANT_NAME>, <APPLICATION_ID>);
```

**Review by 0xnm — APPROVED** [review]
lgtm, I've added some formatting suggestions

**Review by xgouchet — APPROVED** [review]

**Review by rtrieu — APPROVED** [review]
Nice!

**Pull request merged**